src/main/kotlin/hu/kirdev/schpincer/model/CircleEntity.kt

```diff
@@ -36,11 +37,13 @@ data class CircleEntity(
         @Column
         @OrderBy("sort DESC")
         @OneToMany(mappedBy = "circle", fetch = FetchType.LAZY, orphanRemoval = true)
+        @JsonManagedReference
```

Comment:
CircleEntity is not available on the API anymore. Rebase the commits on master.

Reply:
CircleEntity exists on master, I don't understand what you mean by this